ua-snap/epa-justice
# Add new variables

This PR updates the CDC PLACES data to the current 2024 version, adds some new CDC PLACES and SDOH variables to the dataset, and also includes low and high confidence intervals for new CDC PLACES variables. (The kidney disease variable is also removed because it is no longer available in the 2024 data.)
closes #14 
closes #4 

To test:

- Check out the `README.md` and `luts.py` to see the new variables.
- Run the `fetch_data_and_export.ipynb` notebook, which will update the `tbl/data_to_export.csv` with new fields and values
- Run the `join_results_to_census_polygons.ipynb` notebook, which will update the `demographics.dbf` part of the shapefile (which is just the tabular data - the geometry / spatial files will be unchanged)
